platform/ui/src/contextProviders/ViewportGridProvider.tsx

```diff
@@ -238,14 +241,15 @@ export function ViewportGridProvider({ children, service }) {
   );
 
   const setLayout = useCallback(
-    ({ layoutType, numRows, numCols, layoutOptions = [] }) =>
+    ({ layoutType, numRows, numCols, layoutOptions = [], newLayout = {} }) =>
       dispatch({
         type: 'SET_LAYOUT',
         payload: {
           layoutType,
           numRows,
           numCols,
           layoutOptions,
+          newLayout,
```

Comment:
newLayout is not used any where in the code, do you need it?

Reply:
Removed